server/lib/src/constants/schema.rs

```diff
@@ -193,6 +193,24 @@ pub static ref SCHEMA_ATTR_BADLIST_PASSWORD: SchemaAttribute = SchemaAttribute {
     ..Default::default()
 };
 
+pub static ref SCHEMA_ATTR_AUTH_SESSION_EXPIRY: SchemaAttribute = SchemaAttribute {
+    uuid: UUID_SCHEMA_ATTR_AUTH_SESSION_EXPIRY,
+    name: "authsession_expiry".into(),
+
+    description: "An expiration time for an authentication session.".to_string(),
+    syntax: SyntaxType::Uint32,
+    ..Default::default()
+};
+
+pub static ref SCHEMA_ATTR_AUTH_PRIVILEGE_EXPIRY: SchemaAttribute = SchemaAttribute {
+    uuid: UUID_SCHEMA_ATTR_AUTH_PRIVILEGE_EXPIRY,
+    name: "privilege_expiry".into(),
```

Comment:
This should use the `enum Attribute.into()`

Reply:
Done :)